src/tests/youtube/Youtube_EME_002.test.js

```diff
@@ -0,0 +1,119 @@
+import {
+  pluginDeactivate,
+  webKitBrowserStartAndResume,
+  setWebKitUrl,
+} from '../../commonMethods/commonFunctions'
+import constants from '../../commonMethods/constants'
+import { AttachToLogs } from '../../commonMethods/remoteWebInspector'
+
+let url =
+  'https://yt-dash-mse-test.commondatastorage.googleapis.com/unit-tests/2017.html?test_type=encryptedmedia-test&command=run'
```

Comment:
We can drop the 2017 test and only support the 2018 one below

Reply:
Updated the URL to 2019